History.md

```diff
@@ -1,5 +1,14 @@
 # Liquid Change Log
 
+## 5.2.1 (unreleased)
+
+### Features
+
+### Fixes
+
```

Comment:
There is no need to add empty sections (especially a Features section for a patch release).  Just add a section when adding the first item for it.

```suggestion
```